h5p-row.js

```diff
@@ -30,12 +30,31 @@ H5P.Row = (function (EventDispatcher) {
     var createHTML = function () {
       // Create wrapper
       wrapper = document.createElement('div');
+      wrapper.classList.add('h5p-row');
```

Comment:
Beware that the same class is added to the container inside the `attach` method. It's probably okay, but could potentially be confusing if trying to apply styles.